core/src/main/resources/defaults/blueprints/hdp30-shared-services-v2.bp

```diff
@@ -0,0 +1,179 @@
+{
+  "tags": {
+    "shared_services_ready": true
+  },
+  "blueprint": {
+    "Blueprints": {
+      "blueprint_name": "hdp30-shared-services-v2",
+      "stack_name": "HDP",
+      "stack_version": "3.0"
+    },
+    "settings": [
+      {
+        "recovery_settings": []
+      },
+      {
+        "service_settings": [
+          {
+            "name": "HIVE",
+            "credential_store_enabled": "false"
+          }
+        ]
+      },
+      {
+        "component_settings": []
+      }
+    ],
+    "configurations": [
+      {
+        "yarn-site": {
+          "yarn.timeline-service.entity-group-fs-store.active-dir": "file:///hadoopfs/fs1/apps/ats/active",
+          "yarn.timeline-service.entity-group-fs-store.done-dir": "file:///hadoopfs/fs1/apps/ats/done",
+          "yarn.log-aggregation.file-formats": "TFile",
```

Comment:
Not sure if it makes a difference, but I was also including `"yarn.log-aggregation.fs-support-append": "false"` besides `yarn.log-aggregation.file-formats` as per https://docs.google.com/spreadsheets/d/1ZKmcGkMehtO8C6nIAU6mn9k3aFuZJ0ygRqBbM8asq3M/edit#gid=0. (In fact, that sheet mentions `yarn.log-aggregation.fs-support-append` only for WL clusters, so it might not be needed for DL.)